### PR TITLE
feat: make --type optional + minor improvements

### PR DESCRIPTION
**Pull request by iTrooz**
Commit descriptions should be self-explanatory

### CODE REVIEW

File: cli/source/subcommands/format.cpp [hunk 1]
```diff
@@ -23,19 +23,19 @@ namespace pl::cli::sub {
         static bool metaInformation = false;
         static u64 baseAddress = 0x00;
 
-        auto subcommand = app->add_subcommand("format");
+        auto subcommand = app->add_subcommand("format", "Executes the given pattern on the given file, and output the pattern data");
```
Comment (WerWolv) [inline cli/source/subcommands/format.cpp:26]: I'd mention here that it outputs the data as some other format

File: cli/source/subcommands/format.cpp [hunk 2]
```diff
@@ -23,19 +23,19 @@ namespace pl::cli::sub {
...
 
         // Add command line arguments
-        subcommand->add_option("-i,--input,INPUT_FILE", inputFilePath, "Input file")->required()->check(CLI::ExistingFile);
+        subcommand->add_option("-i,--input,INPUT_FILE", inputFilePath, "Input file to parse")->required()->check(CLI::ExistingFile);
```
Comment (WerWolv) [inline cli/source/subcommands/format.cpp:29]: Does it really parse the binary file? Isn't there a better word for this?

File: cli/source/subcommands/run.cpp [hunk 3]
```diff
@@ -22,7 +22,7 @@ namespace pl::cli::sub {
         auto subcommand = app->add_subcommand("run");
 
         // Add command line arguments
-        subcommand->add_option("-i,--input,INPUT_FILE", inputFilePath, "Input file")->check(CLI::ExistingFile);
+        subcommand->add_option("-i,--input,INPUT_FILE", inputFilePath, "Input file to parse")->check(CLI::ExistingFile);
```
Comment (WerWolv) [inline cli/source/subcommands/run.cpp:25]: Same as above

File: cli/source/subcommands/format.cpp (hunk 2 above)
Comment (iTrooz) [reply cli/source/subcommands/format.cpp:29]: I think "parse" is fine, but we could also say "input file to extract data from"

Comment (WerWolv) [reply cli/source/subcommands/format.cpp:29]: I like that more tbh

Comment (iTrooz) [reply cli/source/subcommands/format.cpp]: Okay, i'll change it !

**Pull request merged**